Dfe.PrepareConversions/Dfe.PrepareConversions.Data/Models/AcademisationApplication/AcademisationApplication.cs

```diff
@@ -0,0 +1,428 @@
+         academiesApplicationSchool.SchoolConversionApproverContactName =
+            academisationApplicationSchool.SchoolConversionApproverContactName;
+         academiesApplicationSchool.SchoolConversionApproverContactEmail =
+            academisationApplicationSchool.SchoolConversionApproverContactEmail;
+         academiesApplicationSchool.SchoolConversionTargetDateSpecified =
+            academisationApplicationSchool.SchoolConversionTargetDateSpecified;
+         academiesApplicationSchool.SchoolConversionTargetDate = academisationApplicationSchool.SchoolConversionTargetDate;
+         academiesApplicationSchool.SchoolConversionTargetDateExplained =
+            academisationApplicationSchool.SchoolConversionTargetDateExplained;
+         academiesApplicationSchool.SchoolConversionReasonsForJoining =
+            academisationApplicationSchool.ApplicationJoinTrustReason;
+         academiesApplicationSchool.SchoolConversionChangeNamePlanned =
+            academisationApplicationSchool.ConversionChangeNamePlanned;
+         academiesApplicationSchool.SchoolConversionProposedNewSchoolName =
+            academisationApplicationSchool.ProposedNewSchoolName;
+      }
+
+      public static Application PopulateOverview(AcademisationApplication academisationApplication,
+         out School academisationApplicationSchool, out ApplyingSchool academiesApplicationSchool)
+      {
+         var academiesApplication = new Application();
+         academisationApplicationSchool = academisationApplication.Schools.FirstOrDefault();
+         academiesApplication.ApplyingSchools = new List<ApplyingSchool>(){ new() };
+         academiesApplicationSchool = academiesApplication.ApplyingSchools.FirstOrDefault();
+         academiesApplication.TrustName = academisationApplication.JoinTrustDetails.TrustName;
+         academiesApplication.ApplicationType = academisationApplication.ApplicationType;
+         academiesApplication.ApplicationId =
+            academisationApplication.ApplicationReference;
+         //academiesApplication.ApplicationLeadAuthorName = academisationApplication.Contributors (we have contributors on academisation but how do we know their the lead) // Paul L - awaiting confirmation
+         academiesApplication.ChangesToTrust = academisationApplication.JoinTrustDetails.ChangesToTrust switch
+         {
+            "yes" => true,
+            "no" => false,
+            _ => academiesApplication.ChangesToTrust
+         };
+         academiesApplication.ChangesToTrustExplained = academisationApplication.JoinTrustDetails.ChangesToTrustExplained;
+         academiesApplication.ChangesToLaGovernance = academisationApplication.JoinTrustDetails.ChangesToLaGovernance;
+         academiesApplication.ChangesToLaGovernanceExplained =
+            academisationApplication.JoinTrustDetails.ChangesToLaGovernanceExplained;
+         return academiesApplication;
+      }
+   }
+
+   public class JoinTrustDetails
```

Comment:
Perhaps these should be moved into separate files. So that this class isn't too big.

Reply:
Done